samcli/commands/pipeline/bootstrap/guided_context.py

```diff
@@ -144,43 +183,181 @@ def _prompt_image_repository(self) -> None:
         else:
             self.create_image_repository = False
 
+    def _prompt_permissions_provider(self) -> None:
+        click.echo("Select a user permissions provider:")
+        click.echo("\t1 - IAM (default)")
+        click.echo("\t2 - OpenID Connect (OIDC)")
+        user_provider = click.prompt("Choice", type=click.Choice((["1", "2"])), show_default=False, default="1")
+        self.permissions_provider = OPEN_ID_CONNECT if user_provider == "2" else "iam"
```

Comment:
nit: why not also use a constant for "iam"?

Reply:
Good catch. Updating it to a constant.